themes/default/content/docs/guides/saml/aad.md

```diff
@@ -113,13 +113,13 @@ download the SAML application's configuration data and then pass that to Pulumi.
 ## Signing into Pulumi using Azure AD
 
 Once your Azure AD application is created, and its configuration data passed to Pulumi, you can now
-sign in to the Pulumi Service using your SAML SSO credentials.
+sign in to the Pulumi Cloud using your SAML SSO credentials.
 
 Navigate to [https://app.pulumi.com/signin/sso/](https://app.pulumi.com/signin/sso/) and enter the
 name of your Pulumi organization. If everything is configured correctly, you should be prompted to
-sign in to your Azure AD instance, and then immediately be redirected back to the Pulumi Service.
+sign in to your Azure AD instance, and then immediately be redirected back to the Pulumi Cloud.
```

Comment:
```suggestion
sign in to your Azure AD instance, and then immediately be redirected back to Pulumi Cloud.
```

Reply:
@jkodroff lets leave this as it was before this pr opened, we can come back later if we establish a standard around the.